core-modules/discovery-workshop/mapping.md

```diff
@@ -0,0 +1,25 @@
+---
+description: >-
+  An exercise intended to help Product Managers find a succinct way for their users to reach their goal.
+---
+
+# Mapping
+
+### Introduction
+
+Mapping serves as a way to make clear the path our users take to find a service, understand it and acheive their goals. This exercise gives a clear indication of where the common steps are for each organisation participating in the Design Sprint. The path should be guided by three categories: Discovery, Learning and Using.
+
+Discovery is how the user finds the service for the first time.
+Learning is the process of understanding the services offered by the charity.
+Using is the steps towards acheiving the goal or at least getting closer to the goal.
+
+### Required Materials
+
+A common goal, [user personas](./user-personas/README.md), post-its, pens
+
+### Workflow
+
+1. On a large Miro board, whiteboard or blank wall place large post-its at either side. On the left hand side list the users of the app (these can be directly referencing User Personas). On the right hand side, list the common goal for the sprint.
+1. Each Product Manager should start at their user and outline the steps required to acheive their goal through the stages of Discovery, Learning and Using.
```

Comment:
Steps should be formed by actions the user takes to get to their goal, such as 'Online search (discovery)', 'Browse available services (learning)', or 'Book an appointment (Using)'.

Reply:
Will add this in and make a new PR!